posts/How-to-add-blog-post/index.qmd

```diff
@@ -0,0 +1,214 @@
+-  `type`: The type of blog post. This can be either a post or a page.
+-  `image`: The image that will be used as the featured image for the blog post.
+-  `citation`: The citation for the blog post.
+
+#
+
+##### Step 4 - Write your blog post
+After the front matter, you can start writing your blog post using the Quarto syntax. The Quarto syntax is a simple markup language that is often used to format text on the web. It extends the basic Markdown syntax with additional features and functionality, such as the ability to add columns, tables, and other advanced formatting options.
+
+ For example, you can use # characters to create headings, - or * characters to create bullet points, and > characters to create block quotes.
+
+Note: If you dont know how to write a blog post using the Quarto syntax, you can check out the [Quarto documentation](https://quarto.org/docs/get-started/) for more information.
+
+If you dont want the blog post to be published immediately, you can add the following field to the front matter:
+
+```yaml
+draft: true
+```
+
+#
+
+##### Step 5 - Add images
+To add images to your blog post, add the image file to the same folder as the `index.md`. You can then reference the image in your blog post using the following syntax:
+
+```markdown
+![](./<image-name>.<image-extension>)
+```
+
+#
+
+##### Step 6 - Commit your changes
+Once you have finished writing your blog post, you can save the file and commit your changes to your forked repository.
+
+#
+
+##### Step 7 - Create a pull request
+- Next, you will need to create a pull request to merge your changes to the main branch. You can do this by navigating to the GitHub repository and clicking on the `New pull request` button.
+
+![](./new_PR_new.png){fig-align="center" width="80%"}
+
+- The branch for the base repository should be set to `main` and the branch for the head repository should be set to the branch that you created in the previous step. Click the `Create pull request` button to create a new pull request.
+
+![](./new_PR1.png){fig-align="center" width="80%"}
+
+- You can then add a title and description for your pull request. Click the `Create pull request` button to create a new pull request.
+
+![](./new_PR2.png){fig-align="center" width="80%"}
+
+The title of your pull request should be a short description of the changes that you made.
+it should follow the format `[blog]: Title of your blog`
+
+Automatically, the PR template will be added to the description of your pull request. You can add more information about your pull request in the description.
+
+#
+
+##### Step 8 - Merge your changes
+Once you have created a pull request, you can wait for your changes to be reviewed and merged to the main branch.
+
+#
+
+### Preview your blog post
+Since we are working directly from the Github account and Quarto is a command line tool, you can not preview the HTML rendered version of your blogpost.
+To see the preview:
+
```

Comment:
Just show how they can preview it from GitHub without the command line!

Reply:
I didn't know it is possible to preview without using the terminal

Reply:
I'll check the documentation on how to do it

Reply:
I used the netlify preview link instead, is this okay?

Reply:
Yes, that's what I meant, thank you!